.rubocop.yml

```diff
@@ -2,6 +2,18 @@ Metrics/LineLength:
   Description: 'Limit lines to 120 characters.'
   Max: 120
 
+Metrics/BlockLength:
+  ExcludedMethods:
+    - describe
+    - context
+
+Metrics/MethodLength:
+  CountComments: false
+  Max: 20
+
+Metrics/AbcSize:
+  Max: 40
+
```

Comment:
Where do these changes come from ? Are they manual tweaks by you to get the existing immagine code past a new set of rubocop algorithms/defaults ?